### PR TITLE
Changeset tests 

### PR DESCRIPTION
**Pull request by WanjikuMac**
Minor fix to #450 

* Industry changeset tests
* Company changeset tests

### CODE REVIEW

File: test/companies/schema/industry_test.exs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  @invalid_industry %{}
+
+  describe "industry changeset " do
+    test "industry/1 validates with the correct data" do
+      changeset = Industry.changeset(%Industry{}, @valid_industry)
+      assert changeset.valid?
+    end
+
+    test "errors out with invalid data" do
+      changeset = Industry.changeset(%Industry{}, @invalid_industry)
```
Comment (zacck) [inline test/companies/schema/industry_test.exs]: @WanjikuMac  do we really need this extra map here why not pass in an empty map inling `%{}`

File: test/companies/schema/industry_test.exs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    Schema.Industry
+  }
+
+  @valid_industry %{
+    name: "Mining industry"
+  }
+  @invalid_industry %{}
+
+  describe "industry changeset " do
+    test "industry/1 validates with the correct data" do
```
Comment (zacck) [inline test/companies/schema/industry_test.exs]: @WanjikuMac  I think the function here is `changeset/2`

File: test/companies/schema/industry_test.exs (hunk 1 above)
Comment (WanjikuMac) [reply test/companies/schema/industry_test.exs]: Yea I can do that instead. Thanks

File: test/companies/schema/company_test.exs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    blog: "Company blog",
+    description: "Description of the company",
+    github: "github account of the company",
+    location: "Amsterdam",
+    name: "Company name",
+    url: "link of the company",
+    industry_id: 12346
+  }
+  describe "company changeset" do
+    test "company/2 validates with the correct parameters" do
```
Comment (tajchumber) [inline test/companies/schema/company_test.exs]: can you add an empty line between the `@valid_company_params` and the `describe` please?

Can you

File: test/companies/schema/industry_test.exs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  alias Companies.{
+    Schema.Industry
+  }
+
+  @valid_industry %{
+    name: "Mining industry"
+  }
+
+
+  describe "industry changeset " do
```
Comment (tajchumber) [inline test/companies/schema/industry_test.exs]: also here too many empty lines between `@valid_industry` and `describe`, there should be only one.

btw if you use VS Code you can install the [ElixirLS](https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls). I find it very usefull as you can configure it to automatically format properly the code on save!

File: test/companies/schema/company_test.exs (hunk 3 above)
Comment (WanjikuMac) [reply test/companies/schema/company_test.exs]: Yes I can

File: test/companies/schema/industry_test.exs (hunk 4 above)
Comment (WanjikuMac) [reply test/companies/schema/industry_test.exs]: @tajchumber I use Intellij mainly, but I will definitely checkout that VS code extension, thank you.

File: test/companies/schema/company_test.exs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  @moduledoc """
+    This module holds the unit tests to the company changeset
+  """
+  use Companies.DataCase
+
+  alias Companies.{
+    Schema.Company
+  }
+
+  @valid_company_params %{
```
Comment (doomspork) [inline test/companies/schema/company_test.exs]: We're using `ex_machina` on this project so you could have used that to generate these parameters for you.

https://hexdocs.pm/ex_machina/ExMachina.Ecto.html#c:params_for/2

**Pull request merged**